These aren't a problem for GNU C++, but clang throws an error.
The fix is from upstream (by Sam Varshavchik).

	-mi

--- libmail/mboxadd.H	2017-03-05 20:28:33.000000000 -0500
+++ libmail/mboxadd.H	2020-10-29 00:01:46.241648000 -0400
@@ -29,5 +29,5 @@
 	FILE *fp;
 
-	ptr<mbox> mboxAccount;
+	mail::ptr<mbox> mboxAccount;
 
 	void reportProgress(size_t bytesCompleted,
--- libmail/smtpfolder.H	2017-03-05 20:28:33.000000000 -0500
+++ libmail/smtpfolder.H	2020-10-29 00:09:19.515780000 -0400
@@ -85,5 +85,5 @@
 class smtpAddMessage : public addMessage {
 
-	ptr<smtp> myServer;
+	mail::ptr<smtp> myServer;
 	smtpInfo myInfo;
 	mail::callback &myCallback;
